javascript/javascript1/week3/week3.js

```diff
@@ -0,0 +1,135 @@
+//username on freecodecamp: Rikkaa
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+const shiftName = names.shift(function (e) {
+  return e !== nameToRemove;
+});
+// Code done
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//When will we be there??
+
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function timefunction(destinationDistance, speed) {
+  return travelInformation.destinationDistance / travelInformation.speed;
+}
+
+const travelTime = timefunction(travelInformation);
+console.log(
```

Comment:
Nice job ! Although I could imagine using string interpolation to make it more concise 🐱‍👓